test/valid-data/annotation-comment/main.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @comment Top level comment
+ */
+export interface MyObject {
+    /**
+     * @comment Property comment
+     */
+    nested: MyNestedObject
+}
+
+/**
+ * @comment Nested object comment
```

Comment:
Should this be @comment or @$comment?

Reply:
So in the output schema the keyword would be `$comment`, but as is the case with the JSDoc tag for `$id` i don't think the user should have to add the $ to the comment, the tool should do that for them